Readme.md

```diff
@@ -2,25 +2,25 @@
 
 [![Build Status](https://travis-ci.org/vurtun/nuklear.svg)](https://travis-ci.org/vurtun/nuklear)
 
-This is a minimal state immediate mode graphical user interface toolkit
+This is a minimal-state immediate-mode graphical user interface toolkit
 written in ANSI C and licensed under public domain. It was designed as a simple
-embeddable user interface for application and does not have any dependencies,
-a default render backend or OS window and input handling but instead provides a very modular
-library approach by using simple input state for input and draw
-commands describing primitive shapes as output. So instead of providing a
+embeddable user interface for applications. It does not have any dependencies,
```

Comment:
@sglass68: This is subjective, but IMO this is better as a single sentence due to the tight semantic connection between the two halves and also to have more [variety of sentence length](https://wordsbyevanporter.com/how-to-use-varied-sentence-lengths-for-better-writing/).